prompt_toolkit/application/application.py

```diff
@@ -1156,15 +1156,16 @@ def suspend_to_background(self, suspend_group: bool = True) -> None:
         if _SIGTSTP is not None:
 
             def run() -> None:
+                signal = cast(int, _SIGTSTP)
```

Comment:
Thanks for the suggestions!

I'd probably prefer to cast immediately at the top of the file, if that works. Something like:
```python
_SIGTSTP = cast(Optional[int], getattr(signal, "SIGTSTP", None))
```

Reply:
Unfortunately, that doesn't solve the error. Seems like mypy is not smart enough to filter it to int after `if _SIGTSTP is not None:`. The way I did was the only way I found that gets rid of this mypy error.